.woodpecker.yml

```diff
@@ -108,7 +108,7 @@ steps:
     environment:
       CARGO_HOME: .cargo_home
     commands:
-      - "! cargo tree -p lemmy_api_common --no-default-features -i diesel"
+      - "! cargo tree -p lemmy_api_common --no-default-features -i diesel -e no-dev"
```

Comment:
This fails because https://github.com/LemmyNet/lemmy/pull/5851 made diesel a dev dependency for that crate. Then to make the check pass on 0.19 requires either removing the test and dev dependency or removing this ci check.